.travis.yml

```diff
@@ -113,7 +116,7 @@ install:
           pip install -q pyglet;
           conda install --yes --quiet wxpython > ${REDIRECT_TO};
         else
-          pip install -q https://bitbucket.org/pyglet/pyglet/get/tip.zip;
+          pip install -q pyglet;
```

Comment:
please combine this with the `pip install -q pyglet` above, they don't need to be separate commands anymore because they are the same